modules.d/90dmsquash-live/dmsquash-live-root.sh

```diff
@@ -145,7 +145,7 @@ do_live_overlay() {
         pathspec="/${live_dir}/overlay-$l-$u"
     elif strstr "$overlay" ":"; then
         # pathspec specified, extract
-        pathspec=${overlay##*:}
+        pathspec=/${overlay##*:}
```

Comment:
I would only prepend / if `pathspec` does not start with /.
With your code, `rd.live.overlay=/dev/disk/by-uuid/XXXX:/path/to/overlay.img` ends with `pathspec=//path/to/overlay.img`